### PR TITLE
Replace Vite project with NextJs

### PR DESCRIPTION
**Pull request by negreirosleo**
This PR replaces the current implementation on Vite for a NextJs implementation.

It doesn't have a lot besides some boilerplate code and the auth.

### CODE REVIEW

File: docker/docker-compose.dev.yml
Comment (dmtrek14) [inline docker/docker-compose.dev.yml:1]: Nit: looks like there are some spacing discrepancies here in the `volumes` section that could be cleaned up.

**Review by dmtrek14 — COMMENTED** [review]
This is a pretty straight-forward drop-in replacement of vite, so there wasn't much that stood out in terms of changes. Everything I have here is nitpicks.

File: docker/docker-compose.dev.yml
Comment (anarute) [reply docker/docker-compose.dev.yml:1]: Actually I think it was wrong before and now it's on the same level as the others.

**Comment by anarute** [comment]
Agree with Danielle on the suggestions, other than that it looks good to me :)

File: docker/docker-compose.dev.yml
Comment (negreirosleo) [reply docker/docker-compose.dev.yml:1]: they had only 1 space of identation and I changed it to 2

**Pull request merged**